survey_form/index.html

```diff
@@ -43,8 +43,87 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form__survey--title">
```

Comment:
Ta klasa powinna nazywać się `form-survey__title`, `form-survey` to `B`lok, a `title` to `E`lement, bloki i elementy separuje się `__`, elementy i modyfikatory separuje się `--`